src/views/FeedList.vue

```diff
@@ -1,7 +1,36 @@
 <template>
   <div class="tips-list">
     <div class="actions">
-      <SendTip />
+      <SendTip :feed="feed" />
+      <div class="feed-category-row">
+        <FilterButton
+          :class="{ active: feed === 'main' }"
+          @click="feed = 'main'"
+        >
+          <IconFeed />
+          <span class="vertical-align-mid">
+            {{ $t('Main') }}
```

Comment:
I was against the wording as translation keys, but actually, it may be better than the current approach:
- don't need to use eslint-disable or to define translation keys for unconfident wording
- don't need to maintain the hierarchy of translation keys (in the Base aepp we spent significant resources for that, in superhero-ui/wallet it is almost not maintained so some translations are duplicated, has outdated paths, or key names)
- updation of English wording automatically invalidates translations to other languages